install/image-references

```diff
@@ -18,3 +18,7 @@ spec:
     from:
       kind: DockerImage
       name: docker.io/openshift/origin-machine-config-daemon:v4.0.0
+  - name: oscontainer
```

Comment:
Naming wise I would probably prefer this to either be `machine-os` or `machine-os-content`, so that it sorts with the machine api.  That means you would publish the tag to `machine-os`.

Reply:
Everything here is in a Kube namespace though right?  Is there something somewhere which displays/aggregates without namespaces?  (Hm, I guess `oc get X --all-namespaces`?)

But sure, can prefix with `machine-`.  As far as the suffix...I've been using the "oscontainer" term elsewhere but...eh.  This particular string won't be highly visible, I'm OK with `machine-os-content`.

Reply:
No, these aren't kube namespaces.  These names are the official "per release payload" names, so think of this as our RPM name.  It will be highly visible in the errata and the oc adm release diff content.  

I don't want to make this RHCoS specific since that means we have to fork origin vs OCP at some point.  If we make it machine-os-content it's clearly owned by the same team that owns machines, and we are unlikely to have 2 or three (or if we do, it would be like channels so `machine-os-content-X`).

Reply:
Naming wise this should be named so that someone looking at the release payload can quickly identify what its purpose is (`machine-config-operator`, `etcd`, `cluster-bootstrap`, `cluster-ingress-operator`, etc)